notifications_utils/recipients.py

```diff
@@ -386,26 +384,36 @@ def validate_local_phone_number(number, column=None):
         raise InvalidPhoneError('Not a valid local number')
 
 
-def validate_phone_number(number, column=None, international=False):
+def validate_phone_number(number, column=None, international=False):  # noqa:   C901
 
     if ';' in number:
         raise InvalidPhoneError('Not a valid number')
 
     if (not international) or is_local_phone_number(number):
         return validate_local_phone_number(number)
 
-    number = normalise_phone_number(number)
+    normalized_number = normalise_phone_number(number)
 
-    if number is False:
-        raise InvalidPhoneError('Not a valid international number')
+    if not normalized_number:
+        try:
+            parsedNumber = phonenumbers.parse(number, region_code)
+            isValidNumber = phonenumbers.is_valid_number(parsedNumber)
+            if not isValidNumber:
+                logging.error('%s: This number is not accpeted', number)
```

Comment:
Please review the [logging document](https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/logging_policy.md#warning). This is expected behavior off the "standard path" (use `warning` here and read the document please). `error` is reserved for problems. 

Additionally, the specific issue is that the number is not valid, be specific. "Not accepted" does not give us a clear picture unless we go into the logs. "Failed to validate parsed number: %s, parsed_number". The variable you are logging is wrong (number instead of parsedNumber).

Please fix naming conventions to match python as well. Python uses [snake case](https://en.wikipedia.org/wiki/Snake_case) for variables.